frameworks/Clojure/pedestal/install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk-amd64
+
+export LEIN_HOME=$IROOT/lein
+
+fw_depends leiningen java
```

Comment:
Please rebase or merge master, and use `java7` instead of Java. See #1512 for details. Also, you can remove the line `export JAVA_HOME` from this file once you've merged master, installing it will set that